### PR TITLE
Language picker

### PR DESCRIPTION
**Pull request by eatyourgreens**
Staging branch URL: https://language-picker.pfe-preview.zooniverse.org

https://language-picker.pfe-preview.zooniverse.org/projects/y-dot-liefting/snapshot-hoge-veluwe?env=production is available in English and Dutch

- [x] Add a simple language picker to projects.
- [x] Style the language picker.
- [x] Reload translations when language changes.
- [x] Let project owners add and remove languages from their projects.
- [x] Add tests.

# Required Manual Testing

- [x] Does the non-logged in home page render correctly?
- [x] Does the logged in home page render correctly?
- [x] Does the projects page render correctly?
- [x] Can you load project home pages?
- [x] Can you load the classification page?
- [x] Can you submit a classification?
- [x] Does talk load correctly?
- [x] Can you post a talk comment?

# Review Checklist

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [x] Can you `rm -rf node_modules/ && npm install` and app works as expected?
- [ ] If the component is in coffeescript, is it converted to ES6? Is it free of eslint errors? Is the conversion its own commit?
- [x] Are the tests passing locally and on Travis?

# Optional

- [ ] Have you replaced any `ChangeListener` or `PromiseRenderer` components with code that updates component state?
- [ ] If changes are made to the classifier, does the dev classifier still work?
- [ ] Have you [resized and compressed](https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/image-optimization) any image you've added?
- [ ] Have you added in [flow type annotations](https://flowtype.org/docs/type-annotations.html)?
- [ ] Have you followed the [Springer guidelines for commit messages](https://github.com/springernature/frontend-playbook/blob/master/git/git.md#commit-messages)?

### CODE REVIEW

**Comment by beckyrother** [comment]
Design feedback:
- [ ] use background color #005d69 (dark teal);
- [ ] color: #fff;
- [ ] padding: .5em 1em;
- [ ] text-align: right
- [ ] margin-left between "Language" and select: 1em;

a_project_ _zooniverse

Overall I worry about the height this adds to the navigation. 
Will this only appear on projects with language options, or will it always be visible?
Another option might be to include it as a drop-down option in the nav bar, and on mobile create a nested dropdown.

**Comment by eatyourgreens** [comment]
>Overall I worry about the height this adds to the navigation.
Would it be ok to move it down into the header section? The layout here is only to quickly show how it should work.

>Will this only appear on projects with language options, or will it always be visible?
It doesn't make sense to show it, to me, if there's only one language available for your project.

**Comment by eatyourgreens** [comment]
I've updated the mock data to include almost all the languages that we had on the old Galaxy Zoo and Penguin Watch projects, so the language menu might be more realistic now.

**Comment by beckyrother** [comment]
> Would it be ok to move it down into the header section?
Yes, definitely. 
And I agree, only show it when there are options. 

Thanks!

**Comment by eatyourgreens** [comment]
I kept those separate because the layout might be different in the narrow and wide contexts, but I had the same thought about duplicating code.

File: app/locales/nl.js [hunk 1]
```diff
@@ -54,6 +54,7 @@ export default {
     miniCourseButton: 'Herstart de minitour'
   },
   project: {
+    language: 'Language',
```
Comment (marten) [inline app/locales/nl.js]: "Taal"

**Comment by eatyourgreens** [comment]
@srallen can you see why the single space between 'Language' and the select menu isn't rendering? I am baffled. https://github.com/zooniverse/Panoptes-Front-End/pull/4677/files#diff-f6bac5cf2f79d0acc13e4ab12016c1b1R26

File: app/locales/nl.js (hunk 1 above)
Comment (eatyourgreens) [reply app/locales/nl.js]: 👍

**Comment by eatyourgreens** [comment]
@srallen I ended up using `word-spacing: 2em` to set the spacing. Should be okay, I think.

**Pull request merged**